### PR TITLE
refactor(installation): English docs for 'installation'

### PR DESCRIPTION
**Pull request by oucb**

### CODE REVIEW

File: en/installation/docker.md [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+```YAML
+configs:
+  brokerUrl: tcp://127.0.0.1:11883
+  inputTopics: $share/group1/#
+  ruleId: __emqx_all
+```
+
+* EMQX and Pulsar are on the same server, using the default configuration without modification
+
+* If not on the same server, it was modified to EMQX intranet address.
```
Comment (Zhiw) [inline en/installation/docker.md]: intranet address -> internal address

File: en/installation/single_node.md [hunk 2]
```diff
@@ -0,0 +1,368 @@
...
+   $ sudo apt-get install -y supervisor
+   ```
+
+5. Nginx Installation
+
+   ```bash
+   $ sudo apt-get install nginx
+   ```
+
+#### Server dependent installation
```
Comment (Zhiw) [inline en/installation/single_node.md]: Installing packages

File: en/installation/docker.md [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+
+* Reloading the Nginx configuration file
+
+  ```bash
+  docker exec -it actorcloud-server nginx -s reload
+  ```
+
+## Running
+
+### ActorCloud Rule Engine running
```
Comment (Zhiw) [inline en/installation/docker.md]: Running ActorCloud Rule Engine

Comment (oucb) [reply en/installation/docker.md]: @Zhiw All resolved

**Pull request merged**